policy/README.md

```diff
@@ -0,0 +1,353 @@
+| `start_time` | time    | O | Beginning time-of-day (hh:mm:ss) when the rule is in effect (default 00:00:00) |
+| `end_time`   | time    | O | Ending time-of-day (hh:mm:ss) when the rule is in effect (default 23:59:59) |
+| `days`       | day[]   | O | Days `["sun", "mon", "tue", "wed", "thu", "fri", "sat"]` when the rule is in effect (default all) |
+| `messages`   | { string:string } | O | Message to rider user, if desired, in various languages, keyed by language tag (see [Messages](#messages))|
+| `value_url ` | URL     | O | URL to an API endpoint that can provide dynamic information for the measured value (see [Value URL](#value-url)) |
+
+### Order of Operations
+
+Rules are ordered most-specific to most-general.  E.g. an “earlier” cap rule would take precedence over a “later” cap rule.  The internal mechanics of ordering are up to the Policy editing and hosting software.
+
+<a name="messages"></a>
+### Messages
+
+Some Policies as established by the Agency may benefit from rider communication.  This optional field contains a map of languages to be (optionally?) shown to the user.  (Unanswered question: shown with what frequency?  At sign-up?  Once per day?)
+
+Language tag values will be per [BCP 47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt).
+
+Example for a decreased speed-limit rule for Venice Beach on weekends:
+
+```
+	"messages": {
+	            "en-US": "Remember to stay under 10 MPH on Venice Beach on weekends!”,
+	            "es-US": "¡Recuerda mantener por debajo 10 millas por hora en Venice Beach los fines de semana!"
+	        },
+```
+
+<a name="value-url"></a>
+### Value URL
+
+An Agency may wish to provide dynamic or global rules, e.g. "Within 300 yards of the stadium, 1000 total extra scooters may be deployed, across all Provider(s)."  In this case, compliance is not computable from the information available to a single Provider.  The Agency would provide a specified endpoint to get the current count of vehicles in the service-area, so that individual Providers could decide whether adding some number to those present can be done.  
+
+There are potential complexities that are as-yet unaddressed, such as the latency between a Provider deciding to enter such an area, and the number of vehicles in the area when the scooters arrive.  As this specification is Alpha, feedback and refinements are welcome.
+
+The payload returned from a GET call to the `value_url` will have the following immutable fields:
+
+| Name         | Type      | R/O | Description |
+| ----         | ----      | --- | ----------- |
+| `value`      | integer   | R   | Value of whatever the rule measures |
+| `timestamp`  | timestamp | R   | Timestamp the value was recorded |
+| `policy_id`  | UUID      | R   | Relevant `policy_id` for reference |
+
+MDS will likely include some sort of streaming mechanism in the upcoming releases that may supplant or replace this mechanism.
+
+<a name="endpoints"></a>
+## Endpoints
```

Comment:
This list seems overly complex. In order to make it easier to cities (who might only have a GIS analyst or similar) to be able to implement this, I'd replace it with  the ability to publish a single policy file with associated info.